src/index.html

```diff
@@ -7,8 +7,69 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap" rel="stylesheet">
+    
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
```

Comment:
Try to align the image inside `a` (add class and some styles), to pass tests:
![image](https://user-images.githubusercontent.com/86923089/216421953-cf610707-6021-4dca-9268-bb6003648950.png)